api_app/analyzers_manager/file_analyzers/elf_info.py

```diff
@@ -0,0 +1,36 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import logging
+
+from elftools.common.exceptions import ELFError
+from elftools.elf.elffile import ELFFile
```

Comment:
can you also leverage elftools to add the parsed data of the ELF file like it has been done for PE_Info?

In that way we can use this analyzer not only to extract the telfhash but also to parse ELF content and show it to the user. Otherwise it is pretty meaningless alone here